sources/PTYSession.m

```diff
@@ -8046,6 +8046,18 @@ - (void)screenRaise:(BOOL)flag {
     }
 }
 
+// Sets current session proxy icon.
+- (void) setProxyIcon:(NSString *)value {
```

Comment:
Remove the space after `(void)`. Also, why not name this `setUserDesiredProxyIcon:` since that's really what it's doing?